### ISSUE TITLE
Inability to Arouse

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7111]
**Preferred term label:**
Inability to arouse

**Synonyms**
inability to stay awake, inability to awaken from sleep

**Definition (free text, please give PubMed ID)**
Inability to arouse in response to one or more external stimuli.

**Parent term (use hpo.jax.org/app)**
HP:0004372 - Reduced consciousness/confusion

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-19

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

**Editors Note (or alternative annotation)**
https://journals.lww.com/nursing/fulltext/2008/08000/documenting_level_of_consciousness.53.aspx

**Comment by pnrobinson** [comment]
I am not sure how this is different from existing terms.
Complete inability to arouse is coma -- https://hpo.jax.org/app/browse/term/HP:0001259
Possibly Lethargy is also appropriate -- https://hpo.jax.org/app/browse/term/HP:0001254

Can you provide a link to the PMID where this is being described?

**Comment by pnrobinson** [comment]
Closing this for now, I do not think this is actually a new phenotype compared to the above suggestions. Please reopen if the new phenotype can be defined as different.
